### PR TITLE
[RHDEVDOCS-5020] Bump versions to 1.9.1

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-5020

Link to docs preview:
https://56452--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html
https://56452--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-pipelines-as-code.html#installing-pipelines-as-code-cli_using-pipelines-as-code

QE review: @ppitonak
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by ppitonak — APPROVED** [review]

**Comment by SNiemann15** [comment]
I just verified the chanbged links they work now.

File: modules/op-installing-pipelines-as-code-cli.adoc
Comment (Srivaralakshmi) [inline modules/op-installing-pipelines-as-code-cli.adoc:15]:
```suggestion
* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/1.9.1/tkn-linux-s390x.tar.gz[Linux on IBM Z Systems and IBM(R) LinuxONE (s390x)]
```

I hope this is correct. If yes, please treat this as a global comment for all similar instances in this PR. Else, kindly ignore my suggestion.

**Review by Srivaralakshmi — APPROVED** [review]
@SNiemann15 Nice work!! Approving, hoping that the answer to my query is No and is in your favor. If it is otherwise, please push the required change and get approval from other reviewers in the user group. Thank you!

File: modules/op-installing-pipelines-as-code-cli.adoc
Comment (SNiemann15) [reply modules/op-installing-pipelines-as-code-cli.adoc:15]: Yes it's correct we have a naming change to IBM zSystems and the IBM(R) that always has to accompany LinuxONE now.   
Once this PR is merged for the 1.9.1 GA I will even add the files to my other PR that does a sweep to add the common attributes. https://github.com/openshift/openshift-docs/pull/56320

**Pull request merged**
